packages/web-app/src/lib/get-case-data.js

```diff
@@ -0,0 +1,22 @@
+const { getData } = require('./api-wrapper');
+
+const getCaseData = (req, res, next) => {
+  const {
+    log,
+    params: { appealId },
+    session,
+  } = req;
+
+  try {
+    if (session && !session.appeal) {
+      log.debug({ id: appealId }, 'Getting existing data');
+      session.appeal = getData(appealId);
```

Comment:
Bit confused by this. It looks like this is populating mock data, but shouldn't it populate something a bit more generic?

Reply:
It's populating mock data for now until we can connect the APIs to the database then it can get the submission data for the appeals/replies as necessary